src/redux/actions/adminFormActions.js

```diff
@@ -1,7 +1,11 @@
 export const ACTION_TYPES = {
   SET_COHORT_NAME: "SET_COHORT_NAME",
-  SET_COHORT_TYPE: "SET_COHORT_TYPE"
-}
+  SET_COHORT_TYPE: "SET_COHORT_TYPE",
+  SET_POST_SUCCESS: "SET_POST_SUCCESS",
```

Comment:
I think we might want to rename these actions: SET_POST_SUCCESS, SET_POST_ERROR, SET_RESET_APP to drop the "SET" from the action names. These were used in the SET_COHORT_NAME, SET_COHORT_TYPE because these were actions that set new input values in the form. 

However, for the ones I've mentioned, we can rename them to POST_ADMIN_FORM_SUCESS or SAVE_ADMIN_FORM_SUCCESS for example, and the other 2 we can rename accordingly based on which option of the 2 (post or save in the name) we choose.